### PR TITLE
testing on build6

### PR DESCRIPTION
**Pull request by chexov**

### CODE REVIEW

File: talespinproject/src/main/java/com/vg/ts/RendersToBBoxesTxtMain.java [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+import javax.imageio.ImageIO;
+import javax.swing.*;
+
+import com.google.common.base.Throwables;
+import com.google.common.io.Files;
+import com.google.common.primitives.Ints;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+public class RendersToBBoxesTxtMain {
```
Comment (zhuker) [inline talespinproject/src/main/java/com/vg/ts/RendersToBBoxesTxtMain.java]: cool

File: talespinproject/src/main/java/com/vg/ts/MaskBBox.java [hunk 2]
```diff
@@ -59,9 +65,9 @@ public void addPixel(int x, int y) {
         for (int y = 0; y < image.getHeight(); y++) {
             for (int x = 0; x < image.getWidth(); x++) {
                 int a = data.get() & 0xff;
-                int r = data.get() & 0xff;
-                int g = data.get() & 0xff;
                 int b = data.get() & 0xff;
+                int g = data.get() & 0xff;
```
Comment (zhuker) [inline talespinproject/src/main/java/com/vg/ts/MaskBBox.java]: cool

File: talespinproject/pom.xml [hunk 3]
```diff
@@ -35,5 +35,11 @@
             <artifactId>hppc</artifactId>
             <version>0.7.3</version>
         </dependency>
+        <!-- https://mvnrepository.com/artifact/com.google.guava/guava -->
+        <dependency>
+            <groupId>com.google.guava</groupId>
```
Comment (zhuker) [inline talespinproject/pom.xml]: why?

File: talespinproject/src/test/java/com/vg/ts/MaskBBoxTest.java [hunk 4]
```diff
@@ -22,6 +22,26 @@
...
+    public void testBboxesFromMasksBuild6() throws Exception {
+        String json = new String(readAllBytes(new File("testdata/2.json").toPath()));
+        RenderDesc renderDesc = new GsonBuilder().create().fromJson(json, RenderDesc.class);
+
+        BufferedImage image = ImageIO.read(new File("testdata/2ColorKey.PNG"));
+        BufferedImage render = ImageIO.read(new File("testdata/2v0.PNG"));
+
+        Map<NameMask, Rectangle> bboxes = MaskBBox.bboxesFromMasks(image, renderDesc.Instances);
+        assertNotNull(bboxes);
+        debug(render, bboxes);
```
Comment (zhuker) [inline talespinproject/src/test/java/com/vg/ts/MaskBBoxTest.java]: not cool (cant do `show(image)` in a testcase)

**Comment by chexov** [comment]
this PR was for checking diff. closing it for now

**Pull request closed without merging**